cinema/serializers.py

```diff
@@ -0,0 +1,23 @@
+from rest_framework import serializers
+
+from cinema.models import Movie
+
+
+class MovieSerializer(serializers.Serializer):
+    id = serializers.IntegerField(read_only=True)
+    title = serializers.CharField(required=True)
+    description = serializers.CharField(required=False)
+    duration = serializers.IntegerField(required=True)
```

Comment:
you can use Meta class and  `fields = '__all__'` attribute, no need to list each field separately